.azure/pipelines/fuzz.yml

```diff
@@ -0,0 +1,24 @@
+variables:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  CI_BUILD_NUMBER: $(Build.BuildId)
+  BRANCH_NAME: $(Build.SourceBranchName)
+  TAG_NAME: $(Build.SourceBranchName)
+  SDK_VERSION: 3.x
+
+schedules:
```

Comment:
`trigger` must also be defined, otherwise each commit will trigger the pipeline, and that is not what I want.